docs/ARCHITECTURE.md

```diff
@@ -319,7 +319,19 @@ There are more things in [Worker](https://github.com/enigmampc/enigma-p2p/tree/m
 # Running the Node
 * TBD
 # Running the tests
-* TBD
+
+```bash
+sudo npm install -g truffle
+git clone git@github.com:enigmampc/enigma-p2p.git
+cd enigma-p2p
+npm install
+npm run test
+```
+
```

Comment:
maybe add a `Troubleshooting` title or something like that? It's not critical...

Reply:
Added, thanks!

Reply:
:)